meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -126,20 +127,25 @@ else
     inform "No need to flash bootloader to a specific device."
 fi
 
+# Mount internal device boot partition to write bootloader config file on it
```

Comment:
I would keep this as on block of code: where you mount, copy splash, bootloader config and unmount